docs/modules/continuous-integration/pages/jetbrains-space.adoc

```diff
@@ -0,0 +1,89 @@
+= Jetbrains Space
+
+You can setup a link:https://www.jetbrains.com/help/space/automation.html[Jetbrains Space] Automation for use.
+
+NOTE: If you're already building with either Maven or Gradle then you might use the
+xref:tools:jreleaser-maven.adoc[] or the xref:tools:jreleaser-gradle.adoc[] instead.
+
+[source,kts]
+[subs="+macros"]
+..space.kts
+----
+// Define project version for general use
+val projectVersion = "{{ run:trigger.git-push.commit }}"
+
+// Define a build step
+job("build") {
+    // Specify container image that contains Java
+    // Can also be a maven image if your project
+    // does not rely on the Maven wrapper
+    container(displayName = "Run mvn install", image = "azul/zulu-openjdk:11") {
+        // Full build
+        shellScript {
+            interpreter = "/bin/bash"
+            content = "./mvnw --batch-mode verify"
+        }
+
+        // Save artifacts for release
+        fileArtifacts {
+            remotePath = "{{ run:number }}/helloworld-1.0.0-SNAPSHOT.jar"
+            localPath = "target/helloworld-1.0.0-SNAPSHOT.jar"
+            onStatus = OnStatus.SUCCESS
+        }
+    }
+}
+
+// Create a release
+job("release") {
+    // limit release to git tags
+    startOn {
+        gitPush {
+            anyTag()
+        }
+    }
+
+    // allow use for environment variables
+    parameters {
+        secret("gpg-passphrase", value = "{{ project:gpg-passphrase }}")
+        secret("gpg-public-key", value = "{{ project:gpg-public-key }}")
+        secret("gpg-secret-key", value = "{{ project:gpg-secret-key }}")
+    }
+
+    // Use the predefined JReleaser Docker image
+    container(displayName = "jReleaser", image = "jreleaser/jreleaser-slim:latest") {
+        // use artifact from build step
+        fileInput {
+            source = FileSource.FileArtifact(
+                "{{ run:file-artifacts.default-repository }}",
+                "{{ run:number }}/helloworld-1.0.0-SNAPSHOT.jar"
+            )
+            localPath = "helloworld.jar"
```

Comment:
I'd suggest adding a note here stating that this file must be in a relative path as expected by the JReleaser configuration

Reply:
added a comment for that.